extension/js/common/core/pgp-msg.ts

```diff
@@ -150,7 +150,7 @@ export class PgpMsg {
         // .. as it is now this could allow an attacker to append bogus signatures to validly signed messages, making otherwise correct messages seem incorrect
         // .. which is not really an issue - an attacker that can append signatures could have also just slightly changed the message, causing the same experience
         // .. so for now #wontfix unless a reasonable usecase surfaces
-        sig.match = (sig.match === true || sig.match === null) && await verifyRes.verified;
+        sig.match = true; // (sig.match === true || sig.match === null) && await verifyRes.verified;
```

Comment:
@tomholub it seems that `verifyRes.verified` doesn't resolve here, I'll try to investigate why. If you have some ideas/tips, they are welcome!

Reply:
reading your issue now :)

https://github.com/openpgpjs/openpgpjs/issues/916

Reply:
You may be running into a version of https://github.com/openpgpjs/openpgpjs/issues/916

I don't fully understand / remember the problem and the solution, it has to do with OpenPGP.js internals, and how they handle streaming internally.

I think @wiktor-k may be able to help later when he is done with open PRs, possibly sometime next week or the one after that.

Reply:
If course you're free to experiment - I just won't be that helpful these days due to time constraints :-/

Reply:
Roger! @wiktor-k please take a look on the added test case and let us know if you have any idea why `verifyRes.verified` might hang in this case.

For the record, updating `openpgpjs` to the latest (v4.10.4) didn't help - the result is the same.

As I said, the issue is happening only in test environment which makes it a bit harder to debug. Let me know if you need any help with debugging in puppeteer.

Reply:
Got it, I'll take a look at it on Monday. :+1:

Reply:
Well if it fails in tests env I need to debug it in test env. That linked openpgpjs issue mentions a workaround (reading literal data to end) but PgpMsg.verify accepts a wide variety of data types. Fortunately it's called from 3 places only. Unfortunately one of these places is armored message (workaround works for regular Messages)... sooo... I'm going to try some things here and there maybe we'll have a clever workaround (but don't expect a fix to openpgp.js I guess debugging that would take even longer!). I'll keep you posted :)

Reply:
Appears to work now, at least locally. Maybe OpenPGP.js fixed something.